pkg/rtc/room.go

```diff
@@ -221,17 +221,17 @@ func (r *Room) Join(participant types.LocalParticipant, opts *ParticipantOptions
 	defer r.lock.Unlock()
 
 	if r.IsClosed() {
-		prometheus.ServiceOperationCounter.WithLabelValues("participant_join", "error", "room_closed").Add(1)
```

Comment:
I'm not sure that `participant_join` or error rate is actually a useful metric here. They are rare, and they are not really actionable.

I think the piece we really focus on is participants connecting to the server.

Reply:
should we remove this? wdyt @boks1971 @matkam @cnderrauber

Reply:
Yeah, I think we can remove all the counters here. These are not problems per se. These are conditions under which participant should not be allowed to join. Maybe logs here, but not stats counters.